apps/nextjs-website/react-components/components/Cards/item.tsx

```diff
@@ -0,0 +1,87 @@
+import { Card, CardContent, Typography, Stack, Link, Box } from '@mui/material';
+import { EIcon } from '../common/EIcon';
+import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import { IItem } from '../../types/Cards/Cards.types';
+import { Title, Body } from '../common/Common';
+
+const Item = ({
```

Comment:
Item seems too generic for a name.

Maybe CardsItem to stay consistent with AccordionItem?